### PR TITLE
Dynamic environments

### PR DESCRIPTION
**Pull request by samhstn**
#84

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,27 @@
-[![Build Status](https://samhstn.com/badge)](https://console.aws.amazon.com/codesuite/codebuild/projects/Push)
+# [samhstn.com](http://samhstn.com)
```
Comment (nelsonic) [inline README.md:1]: Are you still configuring the DNS for https://samhstn.com ? 💭 
image

Hope your evening is going well. 😉

Comment (samhstn) [reply README.md:1]: Hey Nelson! How are you? I'd love to have a catch up soon if you're game!

Yes, it's still in progress 😬I'm looking to prioritize this over the coming weeks though

Feel free to unwatch if these notifications become too frequent btw - there may be quite a few incoming.

I can assign when ready for review if you'd like - let me know what works for you!

Comment (nelsonic) [reply README.md:1]: "_unwatch_" ...? _never_! I learn so much from watching your work! 😍
always up for catching up! looking forward to seeing your site when you re-launch it! 👍

**Comment by samhstn** [comment]
Closing in favour of #87

**Pull request closed without merging**